addons/point_of_sale/models/pos_config.py

```diff
@@ -643,12 +643,15 @@ def post_install_pos_localisation(self, companies=False):
             pos_configs = self.search(domain)
             if not pos_configs:
                 self = self.with_company(company)
-                pos_configs = self.env['pos.config'].create({
-                'name': _('Shop'),
-                'company_id': company.id,
-                'module_pos_restaurant': False,
-            })
-            pos_configs.setup_defaults(company)
+                if picking_type_id := self._default_picking_type_id():
```

Comment:
Why wouldn't there be a picking type?

Reply:
Good Question!

`stock` module inherits [`res.company`].`create` which creates `stock.warehouse` for the company but the super is triggered first which installs the `l10n` module and  `_get_picking_type_create_values` method gets triggered later on it.

Should we change the way `pos.config` is getting created? :thinking: